spot-contracts/contracts/BondIssuer.sol

```diff
@@ -4,10 +4,15 @@ pragma solidity ^0.8.17;
 import { OwnableUpgradeable } from "@openzeppelin/contracts-upgradeable/access/OwnableUpgradeable.sol";
 import { EnumerableSetUpgradeable } from "@openzeppelin/contracts-upgradeable/utils/structs/EnumerableSetUpgradeable.sol";
 
+import { BondHelpers } from "./_utils/BondHelpers.sol";
+
 import { IBondFactory } from "./_interfaces/buttonwood/IBondFactory.sol";
 import { IBondController } from "./_interfaces/buttonwood/IBondController.sol";
 import { IBondIssuer } from "./_interfaces/IBondIssuer.sol";
 
+/// @notice Expected at least one mature bond.
+error NoMaturedBonds();
```

Comment:
If we consider that reverting is part of the interface definition (see below), then this could go in the interface file instead of here.